src/vs/workbench/browser/media/style.css

```
@@ -220,6 +220,10 @@ body.web {
 	opacity: 1 !important;
 }
 
+.monaco-workbench input[type="checkbox"]:focus {
+	outline-offset: 2px !important;
```

I'd remove the `!important` flag as it's not needed:

```suggestion
	outline-offset: 2px;
```

---

Fixed it. Thanks!